### PR TITLE
Feature : rtl mode (with options)

### PR DESCRIPTION
**Pull request by Shnoulle**
# Pull Requests

- [x] JSFiddle https://jsfiddle.net/tvf4gt4g/ + https://jsfiddle.net/t7ccex4a/1/
- [x] Github issue https://github.com/seiyria/bootstrap-slider/issues/678
- [x] documentation updates to README file
- [x] examples within [/tpl/index.tpl](https://github.com/seiyria/bootstrap-slider/blob/master/tpl/index.tpl)
- [x] grunt test : Done, without errors.

## Feedback
- [x] rewrite using ES6 template string
- [x] refactor imbricated ternary test
- [x] RtlOptionsSpec unit test

### CODE REVIEW

**Comment by seiyria** [comment]
Definitely a good start, however:

* It needs to pass our unit tests
* You need to add some unit tests to this

**Comment by Shnoulle** [comment]
OK, units test passed (and errors fixed)

2 questions

1. About 'default' : rtl : bool|null : null (or not set) => default ?
2. Unit test : only some ? Add one rtl (forced) everywhere ?

**Comment by seiyria** [comment]
not set is a fine default since it's falsy. as for unit tests, you'd have to ask @rovolution specifically on what he'd like to see with unit tests.

File: src/js/bootstrap-slider.js [hunk 1]
```diff
@@ -384,16 +384,23 @@ const windowIsDefined = (typeof window === "object");
 				this.options[optName] = val;
 			}
 
+			// Check options.rtl
+			if(this.options.rtl==='auto'){
```
Comment (Shnoulle) [inline src/js/bootstrap-slider.js]: Alternative

```
			if (this.options.rtl === null) {
				this.options.rtl = window.getComputedStyle(this.element).direction === 'rtl';
			}
```

File: src/js/bootstrap-slider.js [hunk 2]
```diff
@@ -829,6 +848,7 @@ const windowIsDefined = (typeof window === "object");
 				tooltip_split: false,
 				handle: 'round',
 				reversed: false,
+				rtl: 'auto',
```
Comment (Shnoulle) [inline src/js/bootstrap-slider.js]: Alt : `rtl: null,`

**Comment by Shnoulle** [comment]
About 'default' : currently bool|string => string==='default' use dir='rtl'.

But null!==false see https://github.com/seiyria/bootstrap-slider/pull/679/commits/6dadef35c6b246cc12aeaf2abfe2080066ffaa71#r94231462

**Comment by Shnoulle** [comment]
@rovolution : thanks for reviewing :) 😃 . Maybe about RtlOptionsSpec i need your help after .

I hope to  work on it before end of the week :).

And thanks a lot for this repo

**Comment by Shnoulle** [comment]
- [x] rewrite using ES6 template string
- [x] refactor imbricated ternary test
- [x] RtlOptionsSpec unit test

Attention : using rtl : 'auto' by default update API (different behaviour in a rtl page) then can make it to false by default then don't break API

File: test/specs/RtlOptionsSpec.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      testSlider = new Slider("#testSlider1", {
+        rtl: true
+      });
+
+      var dirIsRtl = $("#testSlider1").siblings().is(".slider-rtl");
+
+      expect(dirIsRtl).toBeTruthy();
+    });
+
+    it("slider use inversed left and right inline style", function() {
```
Comment (rovolution) [inline test/specs/RtlOptionsSpec.js]: great tests!!!!

**Comment by rovolution** [comment]
Just pulled down and ran locally and everything works great! merging now. Thanks again for this awesome PR! Will be available in `v9.6.0` once I merged/publish to NPM

**Comment by rovolution** [comment]
merged and published to NPM at `v9.6.0`

**Comment by Shnoulle** [comment]
Thanks to you :)

**Pull request merged**